src/modules/core/components/ForceActionToggle/ForceActionToggle.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { Toggle } from '~core/Fields';
+import Icon from '~core/Icon';
+import { Tooltip } from '~core/Popover';
+import styles from './ForceActionToggle.css';
+
+const ForceActionToggle = () => {
+  return (
+    <div className={styles.forceContainer}>
+      <FormattedMessage id="tooltip.forceActionLabel" />
```

Comment:
there's a good practice to use default messages as well, so translation files could be auto-generated, but if you're doing like that in this project, it's ok :-)